qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -69,9 +72,21 @@ class MapboxNavigationViewCustomizedActivity : AppCompatActivity() {
             showCustomViews.value = showCustomViews.value?.not()
         }
 
+        binding.dayMode.setOnClickListener {
+            toggleTheme(AppCompatDelegate.MODE_NIGHT_NO)
+        }
+
+        binding.nightMode.setOnClickListener {
+            toggleTheme(AppCompatDelegate.MODE_NIGHT_YES)
+        }
```

Comment:
nit: Would be nice to have a single button switch here

Reply:
Planning on integrating an actual switch. so a single button works with text "Toggle night mode" https://github.com/mapbox/mapbox-navigation-android/pull/5594

Reply:
Done.